about.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about-style.css">
+    <title>Web3 Teams</title>
+</head>
+
+<body>
+
+    <section class="mixed">
+        <header>
+            <nav class="mobile-nav">
+                <div class="mobile-header">
+                    <a href="#"> <img src="./images/hamburger-icon.png" alt="hamburger icon" id="open-btn"></a>
+                </div>
+            </nav>
+
+            <!-- dropdown -->
+            <div class="side-nav" id="side-nav">
+                <a href="#" class="close-btn" id="close-btn"><img src="./images/cross-button.png"
+                        alt="cross button"></a>
+                <ul>
+                    <li>
+                        <a href="./index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="./about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Program</a>
+                    </li>
+                    <li>
+                        <a href="#">Join</a>
+                    </li>
+                    <li>
+                        <a href="#">Sponsor</a>
+                    </li>
+                </ul>
+            </div>
+
+            <!-- nav for desktop -->
+
+            <nav class="top-nav">
+                <ul>
+                    <li>
+                        <a href="#"><img src="./images/fb-icon.png" alt="twitter icon"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="./images/twitter.png" alt="twitter icon"></a>
+                    </li>
+                    <li>
+                        English
+                    </li>
+                    <li>
+                        My Page
+                    </li>
+                    <li>
+                        Logout
+                    </li>
+                </ul>
+            </nav>
+            <nav class="desktop-nav">
+                <a href="./index.html">
+                    <img src="./images/logo.png" class="brand-logo" alt="brand logo">
+                </a>
+                <div class="desktop-nav-items">
+                    <ul>
+                        <li>
+                            <a href="./about.html">About</a>
+                        </li>
```

Comment:
- [ ]  I think it is good to have `Home link` instead of `About link` in About page; this will show that you're more professional. 😄